### PR TITLE
Onboard repo to semaphore

### PR DESCRIPTION
**Pull request by hk10111**
Enable semaphore in service.yml

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -3,3 +3,10 @@ lang: unknown
 lang_version: unknown
 git:
   enable: true
+github:
```
[the remark this answers is not in the document]
Comment (hk10111) [reply service.yml:6]: yes

[the remark this answers is not in the document]
Comment (wadhwa1) [reply service.yml:6]: do we need both?

[the remark this answers is not in the document]
Comment (hk10111) [reply service.yml:6]: github.enable defaults to true anyways, we can remove it

**Review by wadhwa1 — APPROVED** [review]
lgtm

**Pull request merged**